website/docs/docs/use-dbt-semantic-layer/sl-architecture.md

```diff
@@ -2,7 +2,7 @@
 title: "dbt Semantic Layer architecture"
 id: sl-architecture
 description: "dbt Semantic Layer product architecture and related questions."
-sidebar_label: "Architecture"
+sidebar_label: "Semantic Layer architecture"
```

Comment:
do we want to use 'Semantic Layer'? I've asked Alexis [here](https://www.notion.so/dbtlabs/WIP-dbt-Semantic-Layer-Customer-FAQ-07a6b875a45e4fe4ab89b15c1f159416) too